File: omegaUp/basicos/MayorDiversion.py
# link: https://omegaup.com/arena/problem/edm2021n1-diversion/
def encontrar_mayor_diversion(juguetes: list) -> int:
    menor = min(juguetes)
    return sum(j for j in juguetes if j != menor)


if __name__ == "__main__":
    cantidad_juguetes = int(input())
    lista_juguetes = [int(input()) for _ in range(cantidad_juguetes)]

File: omegaUp/basicos/MayorDiversion.java
import java.util.Scanner;

public class MayorDiversion {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = scanner.nextInt();
        int sum = 0;
        int menor = 101;
        
        for (int i = 0; i < number; i++) {
            int diversion = scanner.nextInt();
            sum += diversion;
            if (diversion < menor)
                menor = diversion;
        }
        
        scanner.close();
        System.out.println(sum-menor);
    }
}